Eatery/Controllers/Eateries/Campus/TabbedPageViewController.swift

```diff
@@ -97,14 +109,22 @@ class TabbedPageViewController: UIViewController {
         pageViewController.didMove(toParentViewController: self)
 
         if !viewControllers.isEmpty {
```

Comment:
Can change to:

    if let vc = viewControllers.first {
        ...
    }

And `viewControllers[0] -> vc`